app/helpers/projects_helper.rb

```diff
@@ -0,0 +1,5 @@
+module ProjectsHelper
+    def format_skills(skills = @tech_skills)
+        skills.map { |skill| {"id": skill.id, "value": skill.name } }
```

Comment:
this is a duplicate of UsersHelper

Reply:
maybe move this to its own helpers file, e.g. SkillsHelper? I think this may have been copy-pasta on my part

Reply:
i moved this method over to the 'application helper' and removed it from projects/users helper. projects need to call the same method for the skills typeahead in the projects form.